README.md

```diff
@@ -1,6 +1,50 @@
-# curriculum-cr-training-mobile-menu
+# Portfolio Project
 
+> This project contains the portfolio worked on during the Microverse Code Reviewer Training.
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> I worked on the Mobile Menu.
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+# Screenshot
+## Mobile View
+
+Width - 375px
+
+![Mobile view](./screenshot.png)
+
+
+![Open Mobile Menu](./screenshot2.png)
+
+# This project was built with:
+
+HTML5
+
+CSS3
+
+Linters
+
+# Version Control System
+
+GIT
+
```

Comment:
- [ ] I will suggest you add procedures or steps to get a local copy of your project and how to setup and run it, as this will make your README look professional and give users an easy ride running your project. You can check the README [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for more details